ra/ra.go

```diff
@@ -963,9 +963,14 @@ func (ra *RegistrationAuthorityImpl) UpdateAuthorization(ctx context.Context, ba
 
 	// Store the updated version
 	if err = ra.SA.UpdatePendingAuthorization(ctx, authz); err != nil {
-		// This can pretty much only happen when the client corrupts the Challenge
-		// data.
-		err = core.MalformedRequestError("Challenge data was corrupted")
+		// Previously this was a MalformedREquestError alongside a comment explaning
```

Comment:
While I'm a big fan of commenting in general, I think we don't need a comment documenting what used to be here, just what is here. And since logging an error is the straightforward, normal thing to do, I don't think this specifically needs a comment.

I'd also change the `%q` in the log to a `%s`, which is how we typically do error interpolation.